src/ng/jsonpCallbacks.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/**
+ * @ngdoc service
+ * @name $jsonpCallbacks
+ * @description
```

Comment:
I'm not sure we are consistent with all services, but could add a `@requires $window`.